app/controllers/hyrax/admin/collection_types_controller.rb

```diff
@@ -4,13 +4,31 @@ class Admin::CollectionTypesController < ApplicationController
       authorize! :manage, :collection_types
     end
 
+    layout 'dashboard'
+    class_attribute :form_class
+    self.form_class = Hyrax::Forms::Admin::CollectionTypeForm
+
     def index; end
 
-    def new; end
+    def new
+      add_breadcrumb t(:'hyrax.controls.home'), root_path
+      add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
+      add_breadcrumb t(:'hyrax.admin.sidebar.configuration'), '#'
+      add_breadcrumb t(:'hyrax.admin.collection_types.index.breadcrumb'), hyrax.admin_collection_types_path
+      add_breadcrumb t(:'hyrax.admin.collection_types.new.header'), hyrax.new_admin_collection_type_path
+      @form = form_class.new
+    end
 
     def create; end
 
-    def edit; end
+    def edit
+      add_breadcrumb t(:'hyrax.controls.home'), root_path
+      add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
+      add_breadcrumb t(:'hyrax.admin.sidebar.configuration'), '#'
+      add_breadcrumb t(:'hyrax.admin.collection_types.index.breadcrumb'), hyrax.admin_collection_types_path
+      add_breadcrumb t(:'hyrax.admin.collection_types.edit.header'), hyrax.edit_admin_collection_type_path
```

Comment:
@jeremyf Are you suggesting it should be `edit_admin_collection_type_path(@collection_type)` ?

I know we typically want to provide a parameter when using an edit_..._path, but with the `add_breadcrumb` examples I see in other controllers a parameter is never used.  And this breadcrumb seems to be working fine as-is.

Other examples like this:
- `add_breadcrumb t(:'hyrax.admin.sidebar.content_blocks'), hyrax.edit_content_blocks_path`
- `add_breadcrumb t(:'hyrax.admin.sidebar.pages'), hyrax.edit_pages_path`